src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -204,6 +204,9 @@ public BasePanel(JabRefFrame frame, BibDatabaseContext bibDatabaseContext) {
         Objects.requireNonNull(bibDatabaseContext);
 
         this.bibDatabaseContext = bibDatabaseContext;
+        DatabaseChangeListener databaseChangeListener = new DatabaseChangeListener(this);
```

Comment:
Why don't you just do registerListener(this)  and then do this? No need to create a new listener?

```java
@Subscribe 
public void listen(BibDatabaseContextChangedEvent event)
{
basePanel.markBaseChanged();
}
```